plugins/examples/custom-settings/build.gradle

```diff
@@ -28,7 +27,8 @@ esplugin {
   noticeFile rootProject.file('NOTICE.txt')
 }
 
-testClusters.integTest {
+testClusters.all {
```

Comment:
Is there a reason this needs to be `all` and not just `yamlRestTest`? Isn't there just one test cluster in play here?

Reply:
I figured the best example would be to use `all` , I think that should be the go-to configuration unless you need task specific config. (given the ease and cache behavior). Not a strong opinion however.